### PR TITLE
Update aks-edge-howto-connect-to-arc.md

### PR DESCRIPTION
**Pull request by mgodfre3**
Added Details abotu SPN Role needed to enable Arc for Kubernetes.

### CODE REVIEW

**Comment by Court72** [comment]
@rcheeran

Can you review the proposed changes? Note that the validation warnings must be resolved before merging.

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: AKS-Hybrid/aks-edge-howto-connect-to-arc.md [hunk 1]
```diff
@@ -42,10 +42,17 @@ Provide details of your Azure subscription in the [**aksedge-config.json**](http
 |`ClusterName` | string | Provide a name for your cluster. The default value is `hostname_cluster`. |
 |`Location` | string | The location of your resource group. Choose the location closest to your deployment. |
 |`SubscriptionId` | GUID | Your subscription ID. In the Azure portal, select the subscription you're using and copy/paste the subscription ID string into the JSON. |
+<<<<<<< Updated upstream
```
Comment (sethmanheim) [inline AKS-Hybrid/aks-edge-howto-connect-to-arc.md:45]: @mgodfre3 Is this supposed to be a comment? The PR can't be merged with these markers visible.

File: AKS-Hybrid/aks-edge-howto-connect-to-arc.md [hunk 2]
```diff
@@ -42,10 +42,17 @@ Provide details of your Azure subscription in the [**aksedge-config.json**](http
...
+|`ResourceGroupName` | string | The name of the Azure resource group to host your Azure resources for AKS Edge. You can use an existing resource group, or if you add a new name, the system creates one for you. |
+|`ClientId` | GUID | Provide the Application ID of the Azure Service Principal to use as credentials. AKS uses this service principal to connect your cluster to Arc. You can use the **App Registrations** page in the Azure Active Directory resource page on the Azure portal, to list and manage the service principals in a tenant. Please be aware the Service Principal Requires the "Kubernetes Cluster - Azure Arc Onboarding" Role at either the Subscription or Resource Group level. More information can be found [Microsoft Entra Identity Requirments for Service Principals](https://learn.microsoft.com/en-us/azure/azure-arc/kubernetes/system-requirements#microsoft-entra-identity-requirements)|
+|`ClientSecret` | string | Provide the password for the service principal.  |
+>>>>>>> Stashed changes
```
Comment (sethmanheim) [inline AKS-Hybrid/aks-edge-howto-connect-to-arc.md:55]: Same comment as above.

**Comment by sethmanheim** [comment]
Accepting "Client ID" note, rejecting everything else (per PM). #please-close

**Pull request closed without merging**